static/app/utils/issueTypeConfig/metricIssueConfig.tsx

```diff
@@ -27,6 +27,7 @@ const metricIssueConfig: IssueCategoryConfigMapping = {
     similarIssues: {enabled: false},
     userFeedback: {enabled: false},
     usesIssuePlatform: true,
+    useOpenPeriodChecks: true,
```

Comment:
do we need to set this to `true` if the BASE_CONFIG already has it?

Reply:
oof good catch, this should only be enabled for metric issues rn